tutorials/appengine-yarn/index.md

```diff
@@ -1,34 +1,39 @@
 ---
-title: Using yarn on App Engine
-description: Learn how to use yarn to manage dependencies for node.js applications on Google App Engine
+title: Using Yarn on App Engine
+description: Learn how to use Yarn to manage dependencies for Node.js applications on Google App Engine flexible environment.
 author: justinbeckwith
-tags: App Engine, yarn, Node.js, npm
+tags: App Engine, Yarn, Node.js, NPM
```

Comment:
Being super picky, but `npm` isn't supposed to be capitalized :)  https://www.npmjs.com/